assets/js/googlesitekit/notifications/components/layout/BannerNotification.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * BannerNotification layout component.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { useDispatch } from 'googlesitekit-data';
+import { CORE_NOTIFICATIONS } from '../../datastore/constants';
+import useNotificationEvents from '../../hooks/useNotificationEvents';
+import Banner from '../../../../components/Banner';
+import LearnMoreLink from '../../../../components/Banner/LearnMoreLink';
+import CTAButton from '../../../../components/Banner/CTAButton';
+import DismissButton from '../../../../components/Banner/DismissButton';
+import { Cell, Grid, Row } from '../../../../material-components';
+import { TYPES } from '../../constants';
+
+export default function BannerNotification( {
+	notificationID,
+	type = TYPES.INFO,
+	title,
+	description,
+	errorText,
+	helpText,
```

Comment:
I had also raised this in my review of the Setup CTA notification layout - but for some reason, we didn't do it this way. We should only destructure props that are being "decorated" or used in this layout component. All other props can be passed straight to `Banner` (via `...props`) without being destructured here.

The benefit of this is that in the future, if we do add more functionality and props to `Banner`, that can directly be "used" by any layout component that internally uses `Banner` and we don't have to update these layouts as well. I have done this for `OverlayNotification` layout:

https://github.com/google/site-kit-wp/blob/6e3b919ba545f731186ea1cef021582a614e347b/assets/js/googlesitekit/notifications/components/layout/OverlayNotification.js#L32-L38